packages/bitgenerators/bitgenerators.v0.1.0/opam

```diff
@@ -0,0 +1,48 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "PRNG bitgenerators for OCaml users"
+description:
+  "A purely functional port of numpy's random bitgenerator interface for OCaml users"
+maintainer: ["Zolisa Bleki"]
+authors: ["Zolisa Bleki"]
+license: "BSD-3-Clause"
+tags: ["scientific computing" "psuedo-random numbers"]
+homepage: "https://github.com/zoj613/bitgenerators"
+doc: "https://zoj613.github.io/bitgenerators/bitgenerators/Bitgen/index.html"
+bug-reports: "https://github.com/zoj613/bitgenerators/issues"
+depends: [
+  "ocaml" {>= "4.14.0"}
+  "dune" {>= "3.14"}
+  "ctypes"
+  "stdint"
+  "odoc" {with-doc}
+  "bisect_ppx" {dev & >= "2.5.0" & with-test}
+  "ounit2" {with-test}
+  "csv" {with-test}
+  "core_bench" {dev & >= "0.16.0"}
+  "testu01" {dev}
+]
+conflicts: ["ocaml-option-bytecode-only"]
```

Comment:
```suggestion
available: arch != "arm32" & arch != "x86_32"
```
My bad, I thought it was a different type of failure (and I could not see the log at that moment)

Reply:
Looks like this fixed the errors, thanks!